File: python/src/Behavioral/Command/DvdCommandNameStarsOn.py
#//DvdCommandNameStarsOn.java - one of two Concrete Commands
define ['Behavioral/Command/CommandAbstract'
],(
CommandAbstract
) ->
	class DvdCommandNameStarsOn extends CommandAbstract
		constructor : (@dvdName) ->
		execute : -> @dvdName.setNameStarsOn()

File: csharp/src/Behavioral/Command/DvdCommandNameStarsOn.cs
//DvdCommandNameStarsOn.java - one of two Concrete Commands

namespace Behavioral.Command {

public class DvdCommandNameStarsOn : CommandAbstract {
   private DvdName dvdName;  
    
   public DvdCommandNameStarsOn(DvdName dvdNameIn) {
       this.dvdName = dvdNameIn;
   }    
	override public void execute() {
       this.dvdName.setNameStarsOn();
   } 
}

}
